__tests__/components/ThumbnailWrapper.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ThumbnailWrapper from '../../client/src/components/ThumbnailWrapper';
+
+describe('ThumbnailWrapper', () => {
+  it('renders an ImageThumbnail for every item in imageUrlArray', () => {
+    // Test length of 2
+    let wrapper = shallow(<ThumbnailWrapper imageUrlArray={['a', 'b']} />);
+    expect(wrapper.find('ImageThumbnail')).toHaveLength(2);
+
+    // Test length of 3
+    wrapper = shallow(<ThumbnailWrapper imageUrlArray={['a', 'b', 'c']} />);
+    expect(wrapper.find('ImageThumbnail')).toHaveLength(3);
+
+    // Test length of 4
+    wrapper = shallow(<ThumbnailWrapper imageUrlArray={['a', 'b', 'c', 'd']} />);
```

Comment:
Not sure if you'd necessarily require separate tests for URL lengths of 2, 3 and 4 (unless there's some logic that processes differently based on the number of URLs received)?